File: day14/Java/SumDigits.java
/**
 * @author MadhavBahlMD
 * @date 08/01//2018
 */

import java.util.Scanner;

public class SumDigits {
    public static int sum (int num) {
        if (num/10 < 1)
            return num;
        else
            return (num%10) + sum(num/10);
    }

    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        System.out.println("/* ===== Sum of digits using recursion ===== */");
        System.out.print("\nEnter a number: ");
        int num = input.nextInt();
        System.out.println("Sum of digits of " + num + " is: " + sum(num));
    }
}

File: day14/Python/sumdigits.py
"""
  @author : vishalshirke7
  @date : 08/01/2019
"""


def sum_of_digits(n):
    if n <= 0:
        return n
    else:
        return (n % 10) + sum_of_digits(n // 10)


print(sum_of_digits(int(input())))
